### DISCUSSION TITLE
Migrating Internal Python Packages From GitLab to Gcloud Artifacts

### DISCUSSION THREAD

**Discussion by giscus** [hemantapkh/hemantapkh.github.io#2] [Posts]
# Migrating Internal Python Packages From GitLab to Gcloud Artifacts

This blog shares the journey of our migration from GitLab registry to Google Cloud Artifacts

https://hemantapkh.com/posts/migrating-python-packages-from-gitlab-registry-to-gcloud-artifacts/

**Comment by ashishsubedi** [comment]
This is very informative as we're also trying to do the same. Thanks!
